.github/workflows/helm.yml

```diff
@@ -11,6 +11,8 @@ on:
       - "main"
     paths:
       - "charts/**"
+  release:
```

Comment:
This file was updated to be able to publish the poc charts. Do we want to keep it ?